src/daemon/devnode.c

```diff
@@ -328,9 +358,9 @@ int mkfwnode(usbdevice* kb){
         fprintf(fwfile, "%04x", kb->fwversion);
         fputc('\n', fwfile);
         fclose(fwfile);
-        chmod(fwpath, S_GID_READ);
+        (void) check_chmod(fwpath, S_GID_READ);
```

Comment:
See comments for `check_chmod` in `_updateconnected`.

I shouldn't have to copy and paste this again and again.